lib/docker.js

```diff
@@ -182,7 +182,13 @@ function createDockerFile() {
         contents += 'ENV HOME=/root/ LINK=g++\n';
     }
 
-    contents += 'ENV IN_DOCKER=1\n';
+    var envCommand = 'ENV IN_DOCKER=1';
+    if (pkg.deploy.env && Object.keys(pkg.deploy.env)) {
+        Object.keys(pkg.deploy.env).forEach(function(envVar) {
+            envCommand += ' ' + envVar + '=' + pkg.deploy.env[envVar];
```

Comment:
Perhaps better to quote the values so as not to cause problems with string containing spaces